deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -57,63 +60,26 @@ spec:
         name: ""
         version: v1
       specDescriptors:
+      - description: Metrics related configuration options
+        displayName: Metrics Configuration Options
+        path: metrics
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:'
```

Comment:
Yea I did all of this before and ripped it out :) As explained above, we need to keep all parameters flat and not a mapping.